crates/templates/deploy.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
```

Comment:
Why is this written in Python and not Kotlin @vmax ?

Reply:
@lolski reposting the explanation from our chat here:

So these are the limitations we have to work with:

Given: we want to have bazel run //grammar:deploy-crate that does the actual deployment (bazel build //grammar:deploy-crate should not do anything)
The usual way to achieve that in Bazel is that your rule:
```
// f = ctx.actions.declare_file(...)
// ctx.actions.expand_template(output=f)
// return DefaultInfo(executable = f, runfiles = [...])
```
Now, the executable returned has several limitations:

1. It will be executed by the operating system directly, therefore:
* it can be a shell script
* in can be anything specifying a system-wide available interpreter as a first line (#!/usr/bin/env python3 as we've always done)
* it can be a binary (I think)

What I conclude from that is that cannot be a JAR - in fact, returning it in executable field would result in a runtime error:
invalid file (bad magic number): Exec format error


2. It has to be produced by the rule itself. Attempting to return the same executable as passed into the rule would notify you of that:
'executable' provided by an executable rule 'deploy_crate' should be created by the same rule.

Reply:
@lolski `deploy_crate` was reimplemented in Kotlin in https://github.com/vaticle/bazel-distribution/pull/322/commits/b762f60478c6d8d53f42977b7d9ce40a316d451f using the approach we discussed (shell wrapper script)